.husky/prepare-commit-msg

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+INPUT_FILE=$1
+START_LINE=`head -n1 $INPUT_FILE`
+
+if ! [[ "$START_LINE" =~ ^[\ []CUM-[0-9]+[]\ ][[:space:]]([A-Za-z0-9]|[[:space:]])*$ ]]
```

Comment:
In order to get nicely formatted, auto-generated release notes, we'll need to follow conventional commits. https://www.conventionalcommits.org/en/v1.0.0/

Therefore a commit message should look something like `feat(CUM-1): added cum` or `chore(CUM-2): removed cum`. All the possible "types" are `feat`, `fix`, `build`, `chore, `ci`, `docs`, `style`, `refactor`, `perf` and `test`.